.buildkite/jobs/pipeline.android_rn_77_old_arch.yml

```diff
@@ -1,9 +1,9 @@
-  - label: ":android::detox: (Old Arch) RN .76 + Android: Tests app"
+  - label: ":android::detox: (Old Arch) RN .77 + Android: Tests app"
     command:
       - "nvm install"
       - "./scripts/ci.android.sh"
     env:
-      REACT_NATIVE_VERSION: 0.75.4
+      REACT_NATIVE_VERSION: 0.77.2
       DETOX_DISABLE_POD_INSTALL: true
       DETOX_DISABLE_POSTINSTALL: true
       ENABLE_NEW_ARCH: false
```

Comment:
So apparently there's both `ENABLE_NEW_ARCH` and `RCT_NEW_ARCH_ENABLED` that mean the same thing 🤪 Let's align in the next round of upgrades 😊